src/main/java/seedu/address/model/book/Rating.java

```diff
@@ -8,7 +8,7 @@
  */
 public class Rating {
 
-    public final Integer rating;
+    public final Integer value;
```

Comment:
use `int` instead of `Integer`

Reply:
It will cause some errors in the following part